docs/flow-setup-tutorial.md

```diff
@@ -0,0 +1,34 @@
+# Flow setup Tutotrial
+
+Flow is a static-type checker for Javascript code
+
+## The steps to install Flow correctly from scratch are
+
+####  Installation tip : install flow in the root folder of zulip-mobile
+
+####  package installer used - yarn
+
+## Installing flow steps
+
+1. install babel compiler with command line interface,
```

Comment:
Why is that needed?

Reply:
The flow doc says this " Babel is a compiler for JavaScript code that has support for Flow. Babel will take your Flow code and strip out any type annotations. " , it's the first step of installing flow.

Reply:
Babel is already installed with React Native